Configurations/10-main.conf

```diff
@@ -17,7 +17,7 @@ sub vc_win64a_info {
                                 asflags   => "/c /Cp /Cx",
                                 asoutflag => "/Fo" };
         } else {
-            $die->("NASM not found - please read INSTALL and NOTES.WIN for further details\n");
+            $die->("NASM not found - make sure it's installed and avaible\n");
```

Comment:
avaible -> available. It might be appropriate to put it as "available on %PATH%"...

Reply:
Sure